zest/releaser/vcs.py

```diff
@@ -318,7 +320,7 @@ def list_files(self):
         works is handy for the vcs.txt tests.
         """
         files = []
-        for dirpath, dirnames, filenames in os.walk('.'):
+        for dirpath, dirnames, filenames in os.walk(utils.BASE):
```

Comment:
Interesting point! I'd argue that the current behavior is to grab whatever change log is "sitting next to" our `setup.py`, so if the `setup.py` is nested then I'd expect to see a nested change log as well. Change logs higher up in the directory structure might be unrelated to the python package.

Take a "monorepo" like [twitter-text](https://github.com/twitter/twitter-text ), for example. It wouldn't be unreasonable to add a `python` subdirectory and a separate change log for the ruby gem. Searching from the repository root with `--package-root=python` would find that unrelated change log.

That said, I don't feel strongly about this so I got rid of it.